src/server/external/cache/implementations/memory.js

```diff
@@ -29,10 +29,15 @@ function MemoryCache () {
     cache = {};
   }
 
+  async function deleteKey(key) {
```

Comment:
Why did you create this new function? Why flushAll could not solve this problem?